lib/fog/kubevirt/compute/models/vm_base.rb

```diff
@@ -43,7 +43,7 @@ def parse_object(object)
             :disks            => disks,
             :volumes          => parse_volumes(spec[:volumes], disks),
             :status           => object[:spec][:running].to_s == "true" ? "running" : "stopped",
-            :interfaces       => parse_interfaces(domain[:devices][:interfaces]),
+            :interfaces       => parse_interfaces(domain[:devices][:interfaces], []),
```

Comment:
Setting `[]` will not update the mac address when fetch servers. For that we should update also https://github.com/fog/fog-kubevirt/blob/master/lib/fog/kubevirt/compute/requests/get_server.rb#L16 which is being called both from get_server and list_servers.

Also, it would be great to validate this with a test (i.e. when mac address is reported and when it doesn't).

Reply:
Make sense, will update